code_challenges/hacker_rank/collections_counter.py

```diff
@@ -0,0 +1,24 @@
+from collections import Counter
+
```

Comment:
Per [pep-8](https://peps.python.org/pep-0008/#blank-lines) 2 blank lines between imports and a function.